src/libs/modulegroups.c

```diff
@@ -1338,15 +1992,75 @@ static void _manage_direct_module_toggle(GtkWidget *widget, GdkEventButton *even
   }
   else
   {
-    gr->modules = g_list_delete_link(gr->modules, found_item);
-  }
+    gr->modules = g_list_delete_link(gr->modules, found_item);
+  }
+
+  _manage_direct_save(self);
+}
+
+static gint _basics_item_find(gconstpointer a, gconstpointer b)
+{
+  dt_lib_modulegroups_basic_item_t *ia = (dt_lib_modulegroups_basic_item_t *)a;
+  return g_strcmp0(ia->id, (char *)b);
+}
+
+static void _manage_direct_basics_module_toggle(GtkWidget *widget, dt_lib_module_t *self)
+{
+  dt_lib_modulegroups_t *d = (dt_lib_modulegroups_t *)self->data;
+  gchar *wid = (gchar *)g_object_get_data(G_OBJECT(widget), "widget_id");
+  if(g_strcmp0(wid, "") == 0) return;
+
+  GList *found_item = g_list_find_custom(d->basics, wid, _basics_item_find);
```

Comment:
really not clear if found_item should be freed or not, the Gtk doc is not clear.

Reply:
found item shouldn't bee freed. `g_list_find_custom` returns pointer to element inside list, check for example https://github.com/GNOME/glib/blob/ce005e83c64d3d4fbf482f4bfd27a8aefc50b719/glib/glist.c#L919-L933 :)